app/static/src/app/wodin.ts

```diff
@@ -0,0 +1,56 @@
+import { createApp } from "vue";
+import Vuex from "vuex";
+import WodinSession from "./components/WodinSession.vue";
+import BasicApp from "./components/basic/BasicApp.vue";
+import FitApp from "./components/fit/FitApp.vue";
+import StochasticApp from "./components/stochastic/StochasticApp.vue";
+import AppHeader from "./components/header/AppHeader.vue";
+import { BasicState } from "./store/basic/state";
+import { FitState } from "./store/fit/state";
+import { StochasticState } from "./store/stochastic/state";
+import { storeOptions as basicStoreOptions } from "./store/basic/basic";
+import { storeOptions as fitStoreOptions } from "./store/fit/fit";
+import { storeOptions as stochasticStoreOptions } from "./store/stochastic/stochastic";
+import { initialiseRouter } from "./router";
+import tooltip from "./directives/tooltip";
+import { AppType } from "./store/appState/state";
+
+declare let appType: AppType;
```

Comment:
```suggestion
declare let appType: AppType;
const { Basic, Fit, Stochastic } = AppType;
```
Might be nice to destructure here and replace all Apptype.Basic (and other apps) to Basic.

Reply:
Nice, I always forget to do cool things with destructuring! :)